client/src/components/Header/HeaderSearch.tsx

```diff
@@ -93,6 +93,11 @@ const HeaderSearch: React.FC = () => {
               background: var(--mi-white);
               width: 100%;
               z-index: 12;
+              word-break: break-word;
+                color: var(--mi-black) !important;
```

Comment:
This loks like prettier didn't run which probably means that the css isn't formatted correctly, which probably means it's not displaying some things

Reply:
Oh yeah weird easy fix